ihmc-humanoid-robotics/src/main/java/us/ihmc/humanoidRobotics/communication/packets/HumanoidMessageTools.java

```diff
@@ -2416,38 +2418,40 @@ public static void checkRobotSide(HumanoidBodyPart bodyPart)
          throw new RuntimeException("Need to provide robotSide for the bodyPart: " + bodyPart);
    }
 
-   public static IntrinsicParametersMessage toIntrinsicParametersMessage(CameraPinholeBrown intrinsicParameters)
+   @Deprecated
+   public static IntrinsicParametersMessage toIntrinsicParametersMessage(Object intrinsicParameters)
```

Comment:
I think these are importantly used? Would be worth switching it to the CameraIntrinsics

Reply:
ihmc-perception/CameraIntrinsics aren't available here and I don't want to add that dependency. These are not used in anything relevant.

Reply:
Should it be removed then?

Reply:
If I remove it, it kind of explodes and I would have to end up deleting a lot more things